tests/unit/steps/login.steps.js

```diff
@@ -191,7 +190,82 @@ defineFeature(feature, test => {
 
     then('I expect that the failed login alert is visible', () => {
       expect(wrapper.vm.$data.failedLogin).toBeTruthy();
+      expect(wrapper.html()).toContain('Login failed');
     });
   });
 
+  test('Login failure due to a connectivity problem', ({given, when, then}) => {
+    let $axios = {};
+    const USERNAME = 'User1';
+    const PASSWORD = 'Pass1';
+
+    given(/^a mock instance of Axios$/, () => {
+      /**
+       * Mock implementation of the 'put' method for the API client library
+       */
+      $axios.put = jest.fn((path, options) => {
+        expect(path).toEqual('/api/v1/user');
+        expect(options.username).toEqual(USERNAME);
+        expect(options.password).toEqual(PASSWORD);
+        let error = {
```

Comment:
Would you expect an HTTP response code if you cannot communicate to the HTTP server?